### ISSUE TITLE
Audio device names switched to hebrew after patch

### ISSUE THREAD

**Issue by Thireus** [toleda/audio_CloverALC#65]
I guess the picture says it all:

![capture d ecran 2016-03-22 a 17 27 55](https://cloud.githubusercontent.com/assets/721759/13961392/5049871c-f054-11e5-8985-358a178248a1.png)

Any idea about what could have caused the device names to be displayed in hebrew after patch?

System information:

- ALC892
- 10.11.4 (15E65)
- Clover r3354 (latest)
- Patch applied from audio_cloverALC-110_v1.0q10.command

Edit: Audio works btw, which is the most important I guess ;).

Edit2: Looks like this issue occurs when double patching (which I certainly did) --> https://github.com/toleda/audio_CloverALC/issues/17

Edit3: Issue still present even after reinstalling the 10.11.4 combo update and applying the patch once :(.

**Comment by toleda** [comment]
See README, click on Problem Reporting (link).

**Comment by adrinux** [comment]
dl;dr: If you extract the kext from the 10.11.4 update with Pacifist it only contains hebrew localization. Do the full combo update again.

---

I can shed some light on this. I had the same problem (ALC888, Clover 3411) Both cloverALC and pikeralpha versions gave the same issue: hebrew named devices.

I had this working on the original fresh install of 10.11.1, updated OSX to 10.11.4, but running the script to re-enable audio didn't work (probably I ran an old script from github checkout instead of downloading latest version first).

So I used Pacifist to extract the AppleHDA.kext from the 10.11.4 combo update and Kext Utility to put it in place. Ran the latest patch and yay working sound but with hebrew device names.

I installed the combo 10.11.4 update on a different disk partition/osx copy so I could compare kexts and its clear why you only get hebrew:
![applehdas](https://cloud.githubusercontent.com/assets/41276/14402894/e9b08746-fe39-11e5-8318-9bb1725946e5.png)

So I think this has nothing to do with the scripts, its just a limited kext from the start.
